src/pages/RepoPage/CoverageOnboarding/OtherCI/OtherCI.tsx

```diff
@@ -32,7 +34,8 @@ function OtherCI() {
   const uploadToken = orgUploadToken ?? data?.repository?.uploadToken ?? ''
   const tokenCopy = orgUploadToken ? 'global' : 'repository'
 
-  const uploadCommand = `codecovcli upload-process -t ${uploadToken}${
+  const apiUrlCopy = config.IS_SELF_HOSTED ? ` ${config.API_URL}` : ''
+  const uploadCommand = `./codecov -u${apiUrlCopy} upload-process -t ${uploadToken}${
```

Comment:
on cloud there should not be a `-u` flag. The `-u` is for specifying self-hosted url